resources/views/checkout/partials/sections/login.blade.php

```diff
@@ -37,7 +37,11 @@
                 </div>
                 <div>
                     <x-rapidez-ct::title.sm>@lang('Welcome back') @{{ $root.user?.firstname }}!</x-rapidez-ct::title.sm>
-                    @lang('Blah blah blah something something')
+                    <span>
+                        @lang('Is this not your account?')
+                        <span class="underline cursor-pointer" v-on:click="logout('/')">@lang('Log out')</span>
```

Comment:
Maybe `/login`is better?

Reply:
With doing this I'm noticing that logging out doesn't appear to clear the cart. https://github.com/rapidez/core/blob/master/resources/js/components/Cart/mixins/GetCart.js#L60 this doesn't appear to get called. I'll make an issue for this for now.